### PR TITLE
Integrate Mandrel 22.3

### PR DESCRIPTION
**Pull request by zakkak**

### CODE REVIEW

File: .github/workflows/build-images.yml [hunk 1]
```diff
@@ -28,6 +28,9 @@ jobs:
           - name: "mandrel builder images"
             modules: "quarkus-mandrel-builder-image"
             args: ""
+          - name: "mandrel builder images 21 and 22.0"
```
Comment (cescoffier) [inline .github/workflows/build-images.yml:31]: nice, I had to do this because the agent didn't have enough disk space.

**Review by cescoffier — APPROVED** [review]
LGTM! Thanks!

File: .github/workflows/build-images.yml (hunk 1 above)
Comment (zakkak) [reply .github/workflows/build-images.yml:31]: I think it's OK to remove some of the images as well, e.g.:

* 21.3.2.0
* 22.0.0.2
* 22.1.0.0

are fairly dated.

**Comment by Eng-Fouad** [comment]
@cescoffier @zakkak No Mandrel 22.3.0 Java 19 image? Thanks.

**Pull request merged**